cfpb/consumerfinance.gov
# Fix bad values in video player iframe

## Removals

- Removed deprecated options and bad values from `iframe`

## Additions

- Added CSS to ensure border maintained same look as before

## How to test this PR

- Go to `coronavirus/mortgage-forbearance-during-covid-19-what-know-what-do/`
- Play video
- Test fullscreen functionality
- Test different screen sizes for non-fullscreen and fullscreen
- Ensure there are no visible differences between this PR and prod
